### PR TITLE
Add support for Notifications

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: Runtime/Game/Requests/NotificationRequests.cs [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+    public class LootLockerNotificationRewardCurrency
+    {
+        /// <summary>
+        /// The date the Currency reward was created.
+        /// </summary>
+        public string Created_at { get; set; }
+        /// <summary>
+        /// The date the Currency reward was last updated.
+        /// </summary>
+        public string Updated_at { get; set; }
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/NotificationRequests.cs:61]: Change created_at and updated_at to Date instead, easier to handle.

File: Runtime/Game/Requests/NotificationRequests.cs [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+    public class LootLockerNotificationRewardProgression
+    {
+        /// <summary>
+        /// The date the Progression Points reward was created.
+        /// </summary>
+        public string Created_at { get; set; }
+        /// <summary>
+        /// The date the Progression Points was last updated.
+        /// </summary>
+        public string Updated_at { get; set; }
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/NotificationRequests.cs:113]: Same comment about the rest of these, will not comment on all of them.

**Review by JohannesLoot — APPROVED** [review]
Minor changes, overall looks great 👍 
PR is good, but have a more broader question:
You're using "read" for the notifications, the API uses acknowledge. Should we change that to be the same here? So instead of read, it's acknowledged?

**Comment by kirre-bylund** [comment]
> Minor changes, overall looks great 👍 PR is good, but have a more broader question: You're using "read" for the notifications, the API uses acknowledge. Should we change that to be the same here? So instead of read, it's acknowledged?

@JohannesLoot, Nah. If you find "acknowledged" or something like it then that is wrong. It was changed to read. See discussion in here: https://discord.com/channels/1154801384189857912/1293850876997406720/1293908486228279338

File: Runtime/Game/Requests/NotificationRequests.cs (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/NotificationRequests.cs]: Fixed

File: Runtime/Game/Requests/NotificationRequests.cs (hunk 2 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/NotificationRequests.cs]: Fixed

**Pull request merged**